### DISCUSSION TITLE
请问有没有定位到某个坐标的方法？或者说是说相机定位到某个城市某个地方的方法

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [zouyaoji/vue-cesium#573] [General]
请问有没有定位到某个坐标的方法？或者说是说相机定位到某个城市某个地方的方法

**Comment by pengwei-v2** [comment]
你好，可以使用飞行去定位关键api是flyTo

**Comment by pengwei-v2** [comment]
以下是我自己封装的方法，如有更好的建议请指出
`// ****************飞行事件*************************

// lon精度,
// lat纬度,
// heiht高度,
// flyTime飞行时间,
// heading摇头角度,
// pitch点头角度,
// roll歪头角度
function flyTo(
  lon,
  lat,
  heiht = 200,
  flyTime = 5,
  heading = 0,
  pitch = -90,
  roll = 0
) {
  viewer.camera.flyTo({
    destination: Cesium.Cartesian3.fromDegrees(lon, lat, heiht), //飞行到达地点
    duration: flyTime, //飞行时间
    orientation: {
      //视觉方向
      heading: Cesium.Math.toRadians(heading), //摇头
      pitch: Cesium.Math.toRadians(pitch), //点头
      roll: Cesium.Math.toRadians(roll), //歪头
    },
  });
}`

**Comment by pengwei-v2** [comment]
对了，我这个是用原生的cesium的api封装的
